docs-chef-io/content/supermarket/supermarket_upgrade.md

```diff
@@ -13,70 +13,144 @@ publishDate = 2022-01-03
     weight = 25
 +++
 
+Chef Supermarket uses the PostgreSQL database. [PostgreSQL 9.6 is EOL](https://endoflife.date/postgresql) and Private Supermarket users should upgrade to [Supermarket 4.x](https://www.chef.io/downloads/tools/supermarket) and migrate to [Postgres 13](https://www.postgresql.org/about/news/postgresql-13-released-2077/).
```

Comment:
We're actually shipping 9.2 right now :(

Reply:
@tas50 It's 9.3 actually.

Reply:
Oh yep 🤦

Reply:
OK